scripts/lib/api/htmlToMd.ts

```diff
@@ -123,6 +123,24 @@ function prepareHandlers(meta: Metadata): Record<string, Handle> {
 
       return defaultHandlers.div(h, node);
     },
+    class(h, node: any): any {
+      return buildMdxComponent(h, node);
+    },
+    property(h, node: any): any {
+      return buildMdxComponent(h, node);
+    },
```

Comment:
Can we combine property and attribute? From an end-user perspective, I think they show up the same. I see that in the conversion pipeline, they are different; but maybe we can have them use the same component still?

Reply:
Good idea! As we discussed, I combined class-exception, attribute-property, and function-method